palace.pyx

```diff
@@ -1055,7 +1055,11 @@ cdef class Source:
 
     # TODO: set direct filter
     # TODO: set send filter
-    # TODO: set auxiliary send
+    def set_auxiliary_send(self, slot: AuxiliaryEffectSlot, send: int) -> None:
+        """Connect the effect slot to the given send path,
+        using the filter properties"""
```

Comment:
Sentence-end period is missing, also the closing triple quote should be on the next line.